element-android/pipeline.yml

```diff
@@ -70,7 +70,7 @@ steps:
 
     commands:
       - "mv vector-gplay-universal-*.apk app.apk"
-      - "docker run -v $(pwd)/app.apk:/app.apk --rm -i exodusprivacy/exodus-standalone -j > exodus.json"
```

Comment:
This change technically mounts more than we need into the exodus container; i prefer without the /app mount as it's not required.

It is what the exodus docs suggest though, so *shrug*